src/lib/is-installed.ts

```diff
@@ -2,16 +2,28 @@ import { app } from 'electron';
 import fs from 'fs';
 import path from 'path';
 
-const appPaths: Record< keyof InstalledApps, string > =
-	process.platform == 'win32'
-		? {
-				vscode: path.join( app.getPath( 'appData' ), 'Code' ),
-				phpstorm: '', // Disable phpSotrm for Windows
-		  }
-		: {
-				vscode: '/Applications/Visual Studio Code.app',
-				phpstorm: '/Applications/PhpStorm.app',
-		  };
+let appPaths: Record< keyof InstalledApps, string >;
+
+if ( process.platform === 'darwin' ) {
+	appPaths = {
+		vscode: '/Applications/Visual Studio Code.app',
+		phpstorm: '/Applications/PhpStorm.app',
+	};
+}
+
+if ( process.platform === 'linux' ) {
```

Comment:
Since we are checking the same value (`process.platform`), what do you think about using `else if` here instead?